source/jobs.c

```diff
@@ -28,15 +28,15 @@
  */
 
 #include <assert.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
 
+#include "core_json.h"
```

Comment:
This should be split to be a "Internal Include" and "External Dependency" section imo

Reply:
Will update in next commit